### PR TITLE
[VSPC-6] When adding a new space link, the rotation the user enters s…

### PR DESCRIPTION
**Pull request by Neehaarika**
When adding a new space link, the rotation the user enters should be reflected in the icon

### CODE REVIEW

**Comment by jdamerow** [comment]
either put the code that removed the id after a link is created in this branch, or reopen when the branch with that code has been merged.

**Comment by Neehaarika** [comment]
ok I will re open this branch, once that code has been merged.

**Comment by Neehaarika** [comment]
Since VSPC-1 has been merged, I am reopening this story for review.

File: vspace/src/main/webapp/WEB-INF/views/staff/spaces/space.jsp [hunk 1]
```diff
@@ -34,7 +34,6 @@ $( document ).ready(function() {
 	$("#addSpaceLinkButton").click(function(e) {
 		$("#bgImage").on("click", function(e){
 		    e.preventDefault();
-		    $("#arrow").remove();
```
Comment (jdamerow) [inline vspace/src/main/webapp/WEB-INF/views/staff/spaces/space.jsp]: doesn't this need to be here? Isn't that the code that makes sure there is no second arrow when clicking on the image a second time?

**Pull request closed without merging**